shared/chat/inbox/index.stories.js

```diff
@@ -471,12 +519,34 @@ const provider = createPropProvider(
     BigTeamHeader: p => {
       return getPropProviderProps(p)
     },
+    BigTeamsDivider: ownProps => ({badgeCount: 5}),
     BigTeamChannel: getPropProviderProps,
     FilterSmallTeam: getPropProviderProps,
     FilterBigTeamChannel: getPropProviderProps,
   }
 )
 
+class Wrapper extends React.Component<any, any> {
+  state = {
+    props: propsInboxExpanded,
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
```

Comment:
Had an issue with displaying the the `BigTeamsDivider` – the internal logic depends on state, which is set on didComponentUpdate. The official storybook solution is to [pass down the desired state via a prop](https://storybook.js.org/basics/faq/#can-i-modify-react-component-state-in-stories), which isn't super great so @thebearjew came up with the solution of wrapping the Inbox in a timeout to ensure that didComponentUpdate fires.

Reply:
we shouldn't have any kind of timing logic in the stories else storyshots might get messed up. better to pass a prop or wrap this in a `if (!__STORYSHOTS__)` check

Reply:
great point, hadn't thought of that!